src/Elastic.Markdown/Myst/CodeBlocks/SupportedLanguages.cs

```diff
@@ -179,6 +179,9 @@ public static class CodeBlock
 		{ "xquery", "xpath, xq, xqm" }, // XQuery
 		{ "yml", "yaml" }, // YAML
 		{ "zephir", "zep" }, // Zephir
+
+		//CUSTOM, Elastic language we wrote highlighters for
+		{ "eql", "" }
```

Comment:
Is this supposed to be esql or eql something different?

Reply:
Nevermind. I just saw that `eql` is actually used docs-content.

Reply:
eql and esql both exists :D